go/pkg/filemetadata/filemetadata_test.go

```diff
@@ -98,12 +104,12 @@ func TestComputeFilesWithXattr(t *testing.T) {
 				t.Fatalf("Failed to create tmp file for testing digests: %v", err)
 			}
 			after := time.Now().Truncate(time.Second).Add(time.Second)
-			defer os.RemoveAll(filename)
+			t.Cleanup(func() { os.RemoveAll(filename) })
 			got := Compute(filename)
 			if got.Err != nil {
 				t.Errorf("Compute(%v) failed. Got error: %v", filename, got.Err)
 			}
-			wantDigest, _ := digest.NewFromString(fmt.Sprintf("%s/%d", tc.name, len(tc.contents)))
+			wantDigest, _ := digest.NewFromString(fmt.Sprintf("%s/%d", mockedHash, len(tc.contents)))
```

Comment:
nit: you can still handle the error here, just that it would be a Fatal error in test setup.

Reply:
Act